native/cocos/renderer/gfx-agent/DeviceAgent.cpp

```diff
@@ -284,23 +285,32 @@ void doBufferTextureCopy(const uint8_t *const *buffers, Texture *texture, const
     for (uint32_t i = 0U; i < count; i++) {
         bufferCount += regions[i].texSubres.layerCount;
     }
-    uint32_t totalSize = sizeof(BufferTextureCopy) * count + sizeof(uint8_t *) * bufferCount;
+    
+    auto alignTo = [](size_t size, uint32_t alignment) {
+        return (size + alignment - 1) / alignment * alignment;
+    };
+    
+    Format format = texture->getFormat();
+    uint32_t dataRequestAlignment = formatSize(format, 1, 1, 1); //block size
```

Comment:
It seems dataRequestAlignment is not necessary. Will removing dataRequestAlignment  cause trouble?